CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 ### App Center Crashes
 
 * **[Fix]** Fix an issue to call Crashes delegate callbacks even though a modal window is being shown on macOS apps.
+* **[Fix]** Fix an issue when React Native SDK did not send wrapperSdk information.
```

Comment:
We should move this to the RN repo changelog. Even if the fix is in native, from customer point of view we fixed RN.

Reply:
Also did we check symbols / stack trace correct after that fix?

Reply:
Added [PR](https://github.com/microsoft/appcenter-sdk-react-native/pull/798) for RN repo

Reply:
<img width="1632" alt="Screenshot 2020-02-03 at 15 04 32" src="https://user-images.githubusercontent.com/53528961/73657674-d1b2f000-46a3-11ea-97ca-3f762b09b618.png">
JS stack is not displayed on the portal, I think because it is sent in the message field. Created a separate bug on the board.

Reply:
No that is normal stack trace.

Reply:
The backend will parse the message string actually, that's how it works in RN.

Reply:
how did you prepare symbols?

Reply:
But I don't see this information on the portal:
<img width="857" alt="Screen Shot 2020-02-04 at 4 20 07 PM" src="https://user-images.githubusercontent.com/53528961/73752687-71d74a80-4772-11ea-8799-02d7e75f8c69.png">

Reply:
its unsymbolicated, how did you prepare and upload symbols for this build?
You need to make sure to include the index.ios.map inside the dsym archive.

Reply:
I've tested it with proper js source map and it works.
<img width="1104" alt="Screenshot 2020-02-05 at 17 25 43" src="https://user-images.githubusercontent.com/10907757/73845866-ec6c9c80-483c-11ea-8da0-0192a7267b40.png">

Reply:
Yes, I did send the source map incorrectly...